src/store/About.js

```diff
@@ -14,24 +16,64 @@ const state = {
       name: "Python",
       value: 26
     }
-  ]
+  ],
+
+  developers: [],
+
+  repositories: []
 };
 
 const mutations = {
   SET_LANGUAGES(state, languages) {
     state.popularLanguages = languages;
+  },
+
+  SET_DEVELOPERS(state, developers) {
+    state.developers = developers;
+  },
+
+  SET_REPOSITORIES(state, repositories) {
+    state.repositories = repositories;
   }
 };
 
 const actions = {
   setPopularLanguages({ commit }, languages) {
     commit("SET_LANGUAGES", languages);
+  },
+
+  async getDevelopers({ commit }) {
+    const developers = await API.getDevelopers().catch(e => {
+      console.error(e);
+
+      return [];
+    });
+
+    commit("SET_DEVELOPERS", developers);
```

Comment:
To be able to wait for this to finish on page load, we need to return a promise.
```suggestion
	return new Promise((resolve) => {
	    commit("SET_DEVELOPERS", developers);
	    resolve();
	})
```

Reply:
This function is already async, there's no need to wrap it up in a Promise object.

Reply:
I didn't notice :ok_hand: